src/services/agent-service.ts

```diff
@@ -94,15 +94,25 @@ export class AgentService {
     if (!this.snapshotService) { return response.json({success: false}) }
 
     const configuration = new ConfigurationService().configuration
+    // trim the string of whitespace and concat per-snapshot CSS with the globally specified CSS
+    const percySpecificCSS = configuration.snapshot['percy-css'].concat(request.body.percyCSS || '').trim() || ''
```

Comment:
The final `|| ''` is redundant since `.trim()` will fail if it's not a string and will otherwise return an empty string.

Reply:
Oh, duh, right.